### PR TITLE
tdenginewriter rebuiled

### PR DESCRIPTION
**Pull request by zyyang-taosdata**
tdenginewriter rebuiled

### CODE REVIEW

File: tdenginewriter/pom.xml [hunk 1]
```diff
@@ -19,11 +19,31 @@
     </properties>
 
     <dependencies>
+
+        <dependency>
+            <groupId>com.alibaba</groupId>
+            <artifactId>fastjson</artifactId>
+            <version>1.2.58</version>
```
Comment (huolibo) [inline tdenginewriter/pom.xml:26]: the latest fastjson version is 1.2.79

**Review by dingbo8128 — CHANGES_REQUESTED** [review]
tdenginewrite.md and tdenginewrite-CN.md should be updated

**Review by huolibo — APPROVED** [review]
LGTM

**Comment by zyyang-taosdata** [comment]
> tdenginewrite.md and tdenginewrite-CN.md should be updated

**Comment by zyyang-taosdata** [comment]
> tdenginewrite.md and tdenginewrite-CN.md should be updated

docs updated

**Review by dingbo8128 — APPROVED** [review]

**Review by sangshuduo — APPROVED** [review]

**Pull request merged**